blockchain/hardhat.config.js

```diff
@@ -5,5 +5,11 @@ module.exports = {
   solidity: "0.8.9",
   paths: {
     artifacts: './artifacts',
+  },
+  networks: {
+    goerli: {
+      url: import.meta.env.GOERLI_ALCHEMY_URL,
+      accounts: [import.meta.env.GOERLI_WALLET_PRIVATE_KEY]
+    },
```

Comment:
I think this should be `proccess.env` since its javascript, `import.meta.env` is for `vite`